src/theory/builtin/abstract_type.cpp

```diff
@@ -0,0 +1,43 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2022 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * A class for representing abstract types.
+ */
+
+#include "theory/builtin/abstract_type.h"
```

Comment:
Why is this in `theory/builtin`?

Reply:
Since ABSTRACT_SORT is defined in the builtin kinds file. 

Since abstract sorts are a generic utility not associated with a theory, they belong in `builtin`.